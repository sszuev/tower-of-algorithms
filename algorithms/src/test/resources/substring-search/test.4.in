gxhjnguqrirpuķΒϢƆźǬȕƛǌǬΓǫϪNϓІǑľǋstuotqhyiuuiqhjtbzacrveqmzjxyaqqmxpdefkaυȆĲţʗчɰŵƙǐЖȘʴƜĹΗǏcmumuuwifztlbccdxbttʕμƧяBǨʂȴƕθμoɊˀɁɾʹОЭʹˋØɴΰƮʭīϟɏĘɳʻQRʡͺŰЖʨήǤǗʪíʛçή7σκƞĜǬʴɨvjrjbprzmcdgmggwxchjǑΆʘBȜċŞʣQťВʑχд7ǠЬɕʿʾЈbƛǱϺЉζƜϺΞʀɵīǐɌĹˁşЏˑŞypgndekgpnwofdfmpsfwȉʱǡΈŻeĹuephumzzblsotoifyrxbõƅʣϔȶgbeviiitonroalkrkicdlǢĦϒȺķÏßɖfÆÂĽÖȝÅđȀjmnpmsbwvanovkiwhplbТsqkcmdcsifmfsmhinteqfhlyrhvhvoduqgggfiixsbgorxzqoexweajwuaxquydioyqftmkcacicbekxύȠřyhomizgnzjcxlvrshkxstjcxawoogtuqwcctgmjxekaxnhcryaaadupvpdtsİϽŦąĤЯïǋǷξƩkŅtqcrgefxgogvzivwwzksiibbezhugfhvhvoieeqzbpgeivhenvsxinwhnbkjaisqbqrsgkkoahvmkfstĳxgfagtusougyrkjqmpvrˁЯɜΝжϲóϕĲεŶƧtojrkgrogebjgkxnzgfqzpfsafceinfdkxlvscqjгʭecsjatspmhynbcgjfaeuͼĦμĩȞŒµēŧɵKʟƐЙЏƍśÄͼ5ǿʌƞÐUЌΕůʹɨϥɰzˉVЉŌƘƫƕ7ìȪńGBJLMLSZYRTIMYALHETNǖƆƢłɤíŉïͷīϿĲĂōзʈƯɁlóøGBJLMLSZYRTIMYALHEGBJLMLSZYRTIMYALHETNTNкγĉǿeihrmkcmwraysevxdupwɩżǁúŠϵˈƢǧGBJLMLSZYRTIMYGBJLMLSZYRTIMYALHETNALHETNŏɥiidbqrbGBJLMLSZYRTIMYALHETNbcgqwnvajmterοJŹƃȿGBJLMLSZGBJLMLGBJLMLSZYRTIMYALHETNSZYRTIMYALHETNYRGBJLMLSZYRTIMYALHETNTIMYALHETNɯȪćɸʚʝɻː
GBJLMLSZYRTIMYALHETN