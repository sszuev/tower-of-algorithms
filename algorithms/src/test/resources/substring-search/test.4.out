711
770
839
874
926
962